common4j/src/main/com/microsoft/identity/common/java/util/SchemaUtil.java

```diff
@@ -288,11 +285,11 @@ public static String getTenantId(@Nullable final String clientInfoString,
     }
 
     public static String getDisplayableId(@NonNull final Map<String, ?> claims) {
-        if (!StringExtensions.isNullOrBlank((String) claims.get(MicrosoftStsIdToken.PREFERRED_USERNAME))) {
+        if (!StringUtil.isNullOrEmpty((String) claims.get(MicrosoftStsIdToken.PREFERRED_USERNAME))) {
```

Comment:
since stringutil is seeing such increased usage, we should rewrite this method to not create an extra object in order to pull this off.